compiler/rustc_ast/src/mut_visit.rs

```diff
@@ -783,10 +783,11 @@ pub fn visit_token<T: MutVisitor>(t: &mut Token, vis: &mut T) {
             *span = ident.span;
             return; // Avoid visiting the span for the second time.
         }
+        token::InterpolatedIdent(ident, _) | token::InterpolatedLifetime(ident) => {
```

Comment:
```suggestion
        token::InterpolatedIdent(ident, _is_raw) | token::InterpolatedLifetime(ident) => {
```
`(mut_)visit.rs` use exhaustive matching everywhere to make sure we don't miss something important when visiting.

Reply:
This suggested change doesn't compile:
```
error[E0408]: variable `_is_raw` is not bound in all patterns
   --> compiler/rustc_ast/src/mut_visit.rs:784:52
    |
784 |         token::InterpolatedIdent(ident, _is_raw) | token::InterpolatedLifetime(ident) => {
    |                                         -------    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ pattern doesn't bind `_is_raw`
    |                                         |
    |                                         variable not in all patterns
```
So I will leave it unchanged.

Reply:
Then it needs to be split into two arms, like in the previous version.

Reply:
ok